_topics/en/discreet-log-contracts.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Discreet Log Contracts (DLCs)
+
+## Optional.  Shorter name to use for reference style links e.g., "foo"
+## will allow using the link [topic foo][].  Not case sensitive
+shortname: dlc
+
+## Optional.  An entry will be added to the topics index for each alias
+# aliases:
+#   - TODO
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Contract Protocols
+  - Privacy Enhancements
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Discreet Log Contracts (DLCs)** are a contract protocol where two or
+  more parties agree to exchange money dependent on the outcome of a
+  certain event as determined by an oracle (or several oracles). After
+  the event happens, the oracle publishes a commitment to the outcome of
+  the event that the winning party can use to claim their funds. The
+  oracle doesn’t need to know the terms of the contract (or even that
+  a contract was made).
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  The transactions creating and settling the contract can be made
+  indistinguishable from many other Bitcoin transactions or they can be
+  executed within an LN channel. This makes DLCs more private and
+  efficient than other known oracle-based contract methods, and it’s
+  arguably more secure as an oracle that commits to a false result
```

Comment:
Perhaps Im missing something here, but I dont see how the indistinguishable-ness of the contract's txs is related to oracle generating clear evidence of fraud?